doc/user/content/releases/v0.56.md

```diff
@@ -5,7 +5,7 @@ released: true
 patch: 1
```

Comment:
```suggestion
patch: 2
```

Reply:
@def-, this is the variable the test suite relies on.

Reply:
I see, then I have to manually skip v0.56.0. Done.